.github/workflows/docker.yml

```diff
@@ -10,12 +10,26 @@ on:
 
 jobs:
   build:
-    name: Build and push Docker image
-    runs-on: ubuntu-latest
+    name: Build and publish both arch
```

Comment:
Each individual subtask here still only applies to one arch

Reply:
done